MarsAutomation/Pages/LaguangePage.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection.Emit;
+using System.Text;
+using System.Threading.Tasks;
+using MarsAutomation.Utilities;
+using Microsoft.Extensions.DependencyModel;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.DevTools.V121.Debugger;
+using OpenQA.Selenium.Support.UI;
+
+namespace MarsAutomation.Pages
+{
+    public class LaguangePage
+    {
+
+        // Add new language record
+        public void addnewlanguage(IWebDriver driver, string languageName, int level)
+        {
+            IWebElement addNew = driver.FindElement(By.XPath("//div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/thead/tr/th[3]/div"));
+            addNew.Click();
```

Comment:
please implement the POM pattern see https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html

Reply:
please fix all pages

Reply:
please think about state --> example before adding language you need to make sure there a no existing languages as this could result in false positive test pass
you will need to think about this for all the scenarios

the state is say that you have a test that is adding a language english, the second time you run this test you should delete any existing language so that the state is reset, this will guarantee that your test does not produce false negatives and passes always
you can assume that test execution order is not always guaranteed
imagine if you run the tests  in parallel mode the all tests will run at the same time
so you need to also make sure you do not wipe out other tests data
hence why when you remove data the best thing to do is delete only data you added for the test to be correct
e.g test 1 added english and french as language test 2 added hindi then when you reset data you already know what language you are adding for each test
there for on reset of the two test you should delete english and french for test 1 and hindi for test 2